user_tools/src/pyrapids/cloud_api/emr.py

```diff
@@ -0,0 +1,556 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation specific to EMR"""
+
+import configparser
+import json
+import os
+from dataclasses import field, dataclass
+from typing import Any
+
+from pyrapids.cloud_api.emr_job import EmrServerlessRapidsJob, EmrLocalRapidsJob
+from pyrapids.cloud_api.s3storage import S3StorageDriver
+from pyrapids.cloud_api.sp_types import PlatformBase, ClusterBase, CMDDriverBase, CloudPlatform, \
+    ClusterState, SparkNodeType, ClusterNode, GpuHWInfo, SysInfo, GpuDevice
+from pyrapids.common.prop_manager import JSONPropertiesContainer, AbstractPropertiesContainer
+from pyrapids.common.sys_storage import FSUtil
+from pyrapids.common.utilities import find_full_rapids_tools_env_key, get_rapids_tools_env, get_sys_env_var
+from pyrapids.pricing.emr_pricing import EMREc2PriceProvider
+from pyrapids.pricing.price_provider import SavingsEstimator
+from pyrapids.rapids.rapids_job import RapidsJobPropContainer, RapidsJob
+
+
+@dataclass
+class EMRPlatform(PlatformBase):
+    """
+    Represents the interface and utilities required by AWS EMR.
+    Prerequisites:
+    - install gcloud command lines (gcloud, gsutil)
```

Comment:
change references from gcloud to aws cli

Reply:
Done!